kernel/Dockerfile

```diff
@@ -33,10 +34,23 @@ ARG KERNEL_SERIES
 ARG DEBUG
 
 ENV KERNEL_SOURCE=https://www.kernel.org/pub/linux/kernel/v4.x/linux-${KERNEL_VERSION}.tar.xz
+ENV KERNEL_SHA256_SUMS=https://www.kernel.org/pub/linux/kernel/v4.x/sha256sums.asc
+ENV KERNEL_PGP2_SIGN=https://www.kernel.org/pub/linux/kernel/v4.x/linux-${KERNEL_VERSION}.tar.sign
 
-RUN curl -fsSL -o linux-${KERNEL_VERSION}.tar.xz ${KERNEL_SOURCE}
+# PGP keys
```

Comment:
Do you want me to change the comment from:
```
# PGP keys
```
To:
```
# PGP keys for 589DA6B1 (greg@kroah.com) and 6092693E (autosigner@kernel.org)
```
Then commit, squash and sign?